tests/experimental/common/test_reducers_and_aggregators.py

```diff
@@ -104,8 +104,8 @@ def test_min_max_mean_reducers(self, reducer_name, ref, reducers):
             assert self.all_close(val[0].tensor, ref[i])
 
     @pytest.mark.parametrize('reducer_name,ref',
-        [('quantile', ([-20000], [10000])),
-         ('abs_quantile', ([20000],))])
+        [('quantile', ([[[[-20000]]]], [[[[10000]]]])),
```

Comment:
Do we really need such a statistics format for `quantile` reducers?

Reply:
This is needed to make `quantile` reducer interchangeable with other reducers like `min`, `mean` and etc